composer.json

```diff
@@ -19,7 +19,7 @@
   },
   "require-dev": {
     "friendsofphp/php-cs-fixer": "3.*",
-    "phpunit/phpunit": "^8.0 || ^9.0",
+    "phpunit/phpunit": "^9.0",
```

Comment:
The schema of xml for phpunit was giving warnings when I installed the php packages using `composer update`. It was causing problems when I was trying to debug the code. So, I ended up updating the xml schema in`phpunit.xml.dist` and upgrading it to 9.0.